backend/lib/adapters/gateways/mongo/consolidations.mongo.repository.test.ts

```diff
@@ -177,54 +175,102 @@ describe('Consolidations Repository tests', () => {
 
   describe('error handling', () => {
     const error = new Error('some error');
-    const camsError = getCamsError(error, 'COSMOS_DB_REPOSITORY_CONSOLIDATION_ORDERS');
 
     test('should properly handle error when calling search', async () => {
       const consolidationOrder = MockData.getConsolidationOrder({
         override: { consolidationId, courtDivisionCode: '081' },
       });
-      jest.spyOn(MongoCollectionAdapter.prototype, 'find').mockRejectedValue(error);
+      vi.spyOn(MongoCollectionAdapter.prototype, 'find').mockRejectedValue(error);
       await expect(() =>
         repo.search({
           divisionCodes: ['081'],
           consolidationId: consolidationOrder.consolidationId,
         }),
-      ).rejects.toThrow(camsError);
+      ).rejects.toThrow(
+        expect.objectContaining({
+          message: 'Unknown Error',
+          status: 500,
+          module: 'CONSOLIDATIONS-MONGO-REPOSITORY',
```

Comment:
Why did the module this error comes from change? It looks like it used to be `COSMOS_DB_REPOSITORY_CONSOLIDATION_ORDERS`.